ports/esp32/machine_pwm.c

```diff
@@ -71,28 +71,31 @@ STATIC ledc_timer_config_t timers[PWM_TIMER_MAX];
 #define TIMER_IDX_TO_MODE(timer_idx) (timer_idx / LEDC_TIMER_MAX)
 #define TIMER_IDX_TO_TIMER(timer_idx) (timer_idx % LEDC_TIMER_MAX)
 
-// Params for PWM operation
+// Params for PW operation
 // 5khz is default frequency
-#define PWM_FREQ (5000)
-// default 10-bit resolution (compatible with esp8266 PWM)
-#define PWM_RES_10_BIT (LEDC_TIMER_10_BIT)
+#define PWFREQ (5000)
+
+// 10-bit resolution (compatible with esp8266 PWM)
+#define PWRES (LEDC_TIMER_10_BIT)
```

Comment:
IMO it's best to leave these macros named as they were to minimise the diff.  Also, as above, "PW" is confusing.

Reply:
As above. Just history.